hotfails/wbtcnanc.dat

```diff
@@ -0,0 +1,51 @@
+QD> (mapcar 'name (markets *binance*))
+QD> (mapcar 'name (assets *binance*))
+QD> (trace http-request) (HTTP-REQUEST)
+QD> (find-asset "WBTC" *binance*)
+#<WBTC on #<BINANCE>>
+QD> (find (find-asset "WBTC" *binance*) (markets *binance*) :key 'primary)
+#<WBTCUSDT on #<BINANCE>>
+QD> (find (find-asset "WBTC" *binance*) (markets *binance*) :key 'counter)
+NIL
+QD> (untrace) T
+QD> (binance::public-request "exchangeInfo" ())
+
+;;    (:|defaultSelfTradePreventionMode| . \"EXPIRE_MAKER\")
+;;    (:|allowedSelfTradePreventionModes| \"EXPIRE_TAKER\"
+;;     \"EXPIRE_MAKER\" \"EXPIRE_BOTH\"))))" :REPL-RESULT)
```

Comment:
consider whose responsibility is the compression that could prevent the failure...

- [ ] SLIME_...?
- [ ] Binance_?
- [ ] Scalpl.,@?

please only check boxes after composing written argument about why/not.